Makefile

```diff
@@ -1,3 +1,10 @@
+build-docs:
```

Comment:
Maybe you are already aware, but you can automatically host docs under `/docs` with github pages by enabling it under the repo settings. There has to be a `docs/index.html` : https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/#publishing-your-github-pages-site-from-a-docs-folder-on-your-master-branch

Reply:
I'm not aware, thanks. This is all internal API documentation so I'm not actually planning to host it anywhere as of yet.